volta_description/urdf/accessories/rpi_a3_volta.urdf.xacro

```diff
@@ -32,7 +32,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
   <xacro:macro name="lidar" params="frame:=lidar topic:=scan_filtered sample_size:=271 update_rate:=50
                  min_angle:=-1.57 max_angle:=1.57 min_range:=0.5 max_range:=10.0 robot_namespace:=/">
       <joint name="lidar_joint" type="fixed">
-          <origin xyz="${lidar_joint_x} ${lidar_joint_y} ${lidar_joint_z}" rpy="0 0 0" />
+          <origin xyz="${lidar_joint_x} ${lidar_joint_y} ${lidar_joint_z}" rpy="0 0 3.14159" />
```

Comment:
By changing this yaw value, it will effect in the simulation as well....so can we work with previous values itself?
Need to cross check and change this